ISISComputingGroup/EPICS-Mezflipr
# Ticket 2745: Removed duplicate fields

https://github.com/ISISComputingGroup/IBEX/issues/2745
to Test:
run https://github.com/ISISComputingGroup/EPICS-DbChecker/pull/6 against master and confirm there are errors in Mezflipr then run against this branch and confirm the Mezflipr errors no longer occur.
